### ISSUE TITLE
Open source?

### ISSUE THREAD

**Issue by joedf** [Rudloff/emoji-everywhere#3]
This file `data/components/twemoji/twemoji.min.js` that actually processes the HTML DOM cannot be found within this repository. I understand that it is from [here](https://github.com/twitter/twemoji/blob/gh-pages/twemoji.min.js). However, I do not understand why it is found here. I've checked their license and it is released under the MIT license. That said, it only requires attribution/(mention of the copyright notice and this permission notice). Usually, `Copyright (c) <year> <copyright holders>` is enough.

I am not sure, but is this project under GPL?
Is there a confilct between these licenses?

**Comment by Rudloff** [comment]
Hello,

The Twemoji library is installed with [Bower](http://bower.io/) (see the [bower.json](https://github.com/Rudloff/emoji-everywhere/blob/master/bower.json) file) and is indeed under the MIT license.
My code (the files in this repository) is under GPL 3.
As the MIT license is GPL-compatible, it is totally legal to use MIT code in a GPL project.

However, this repository was missing a license notice so I added one on the develop branch: https://github.com/Rudloff/emoji-everywhere/commit/451d10e6dce65a99292ce8bca37e6524bfb9c3dd
I also clarified the license in the README: https://github.com/Rudloff/emoji-everywhere/commit/fed8f32fe535137cdbc471207cfc01ee43c99c2b
